docs/appkit/react/ethers/hooks.mdx

```diff
@@ -1,19 +1,5 @@
 import Button from '../../../components/button'
 
-### useAppKitAccount
-
-Hook that returns the client's information.
-
-```tsx
-import { useAppKitAccount } from '@reown/base/react'
-
-function Components() {
-  const { address, status, isConnected } = useAppKitAccount()
-
-  //...
-}
-```
-
```

Comment:
This hook exists on `@reown/appkit/react`. Any particular reason why we are removing this? 

It also exists on the latest `1.1.0` version of AppKit. Can you please confirm and add it back?